src/Form/atoms/InputFileButton.tsx

```diff
@@ -45,6 +45,7 @@ const InputFileButton : React.FC<IFileInput> = ({text, buttonDesign, buttonSize,
         ref={fileInputRef}
         type='file'
         onChange={handleFile}
+        accept='.jpeg, .png, .csv'
```

Comment:
This `.jpeg, .png, .csv` should not be hardcoded here. It should be passed through props. 
But IMO, As ...props is passed to HiddenInput component so `accept` will be passed by default in props. Have you checked this without making this change?

Reply:
Requested changes are implemented, kindly review